testdata/README.md

```diff
@@ -2,6 +2,24 @@
 
 This directory contains cryptographic artifacts necessary for the Bootz implementation. The files here are included as examples only and real implementations would need their own method of generating, storing and retrieving these.
 
-## ca.pem
+## Regenerating test data
 
-This is an x509 certificate that represents the device manufacturer's root (or some intermediate) CA. It is required that the ownership voucher passed from the Bootz server is signed by this CA.
\ No newline at end of file
+All certificates, private keys and OVs can be regenerated by running the `generate` binary. Ensure all text files are deleted from this directory before running to avoid conflicts.
```

Comment:
It'd be useful to mention that these security artifacts should only be used for testing and MUST NOT be used in any production setup.